### PR TITLE
WIP, DO NOT MERGE page: print actual consumed bytes in a page

### PR DESCRIPTION
**Pull request by mitake**
@ahrtr It seems that current bbolt command doesn't have a functionality of printing actually consumed bytes in a page (sorry if I'm missing). It might be useful if we can print such data in the command e.g. for understanding how FillPercent affects storage utilization. What do you think? If you think it's useful, I'll enhance this branch for actually usable.

Example output:
```
Page ID:    13
Page Type:  leaf
Total Size: 16384 bytes
Consumed Size: 14701
Overflow pages: 0
Item Count: 241

00000000000003cb5f0000000000000000: 0a080000000000000000100418cb0720c8072a08de7eb8ac06c26b6d
00000000000003cc5f0000000000000000: 0a080000000000000000100418cc0720c9072a08de7eb8ac06c26b6d
00000000000003cd5f0000000000000000: 0a080000000000000000100418cd0720ca072a08de7eb8ac06c26b6d
```

`Consumed Size: 14701` is a newly added line.

### CODE REVIEW

**Comment by ahrtr** [comment]
Yes, it's useful, and I am OK to add it. 

Thanks @mitake

**Comment by cenkalti** [comment]
@mitake Is this calculation is the same with the calculation when `bbolt stats <db>` command run?

**Comment by mitake** [comment]
@cenkalti ah yeah `bbolt stats` does similar thing, let me unify the logic for avoiding duplicated code. I think it's useful if `bbolt stats` can show some additional stats of utilization like min, max, percentile, etc of pages in a db file. What do you think?

**Review by cenkalti — CHANGES_REQUESTED** [review]

File: cmd/bbolt/page_command.go [hunk 1]
```diff
@@ -142,6 +146,26 @@ func (cmd *pageCommand) PrintMeta(w io.Writer, buf []byte) error {
...
 }
 
+func consumedSizeOfLeaf(buf []byte) int {
+	p := common.LoadPage(buf)
+	consumedSize := 0
+
+	for i := uint16(0); i < p.Count(); i++ {
+		e := p.LeafPageElement(i)
+
+		consumedSize += int(unsafe.Sizeof(*e))
```
Comment (cenkalti) [inline cmd/bbolt/page_command.go:156]: You can use `common.LeafPageElementSize` instead of using `unsafe` package.

**Pull request closed without merging**